Create whats-new-in-v72.rst (#5821)
mattermost/docs: source/welcome/whats-new-in-v72.rst

Boards is moving from a channel-based to a role-based permissions system. This means that access to individual boards can be controlled on a channel level or a board level, depending on whether a board is linked to a channel or not.

TL;DR
------

1. All the boards you're currently a member of from your current team will appear on the sidebar without needing to switch workspaces.
2. Organize boards on the sidebar with custom categories. 
3. Press Ctrl+K / Cmd+K to find additional boards.
4. Navigate between teams in Boards with the new team switcher.
5. Set board and template permissions in the new **Share** dialog.
6. Link boards to channels to automatically grant board permissions to channel members.

Navigation
----------

Sidebar categories
~~~~~~~~~~~~~~~~~~~

Navigation is simplified, and now works similarly to channels. No more switching between workspaces - and the dashboard page has been removed. All the boards you're currently a member of from your current team will appear on the sidebar without needing to switch workspaces. Use the search box (Ctrl+K / Cmd+K) to find additional boards.

> .. image:: ../images/boards-whats-new-01.png

Reviewer: @justinegeffen Can we reduce the width of this image to 303px so it's the same width as the 2nd image after this one?

Author: Sure. Let me know if that looks better.

.. image:: ../images/boards-whats-new-04.png
  :alt: You can find the boards icon in the Apps Bar on the right-hand side of the screen.

Channel members can search and link boards within the team where they are also board admins. Linking a board to a channel automatically grants all channel members Editor access to the board.

.. note:: 
  
  A channel can be linked to multiple boards, but each individual board can only be linked to one channel at a time. Linking the same board to another channel will automatically remove the link from the previous channel.

Channel members can also create a new board from the RHS. Doing so will automatically link the new board to the channel and grant channel members permissions to the board.

.. image:: ../images/boards-whats-new-05.png
  :alt: You can create a board from the RHS which is automatically added to the channel you're in.

Sharing
-------

Select **Share** to view and edit permissions to a board.

.. image:: ../images/boards-whats-new-06.png
  :alt: Select the share button to open up the permissions options.

> 

Reviewer: Can we force a line break between these 2 images?

Author: Let me know if it looks better?

Direct messages (DMs) and group messages (GMs)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

If a board or custom template was previously attached to a direct message or group message: It's now searchable by, and accessible to, any member of that channel. Future members will need to be added by the admin.

For boards previously attached to DMs and GMs whose members are associated with **more than one team**:

- The board will be moved to the first team on the team sidebar where all the board members are current team members.
- In the case where some members do not belong to the same team, we will associate the board to the first team where the creator (board admin) has access and where most other board members are current team members.

.. note::
   
   These boards may not appear in your search results when switching teams.

Workspaces to categories
~~~~~~~~~~~~~~~~~~~~~~~~

If you belonged to a workspace at the time of migration you’ll see that they’ve been migrated to custom categories in the sidebar. All boards from a workspace are listed under a category of the same name. Boards from direct messages and group messages appear under the default **Boards** category.

> Categories are per-user, and can be renamed or deleted by each user after migration. New users won’t have default categories, and boards they join will appear under the default **Boards** category.

Reviewer: ```suggestion
Categories are per-user, and can be renamed or deleted by each user after migration. New users won’t see the custom categories created from migration, and boards they join will appear under the default **Boards** category.
```

> Boards that you create after the migration won’t be linked to a workspace. and will always appear under the default **Boards** category unless you move or hide the boards.

Reviewer: ```suggestion
Boards that you create after the migration won’t be linked to a workspace, and will always appear under the default **Boards** category unless you move or hide the boards.
```

Frequently asked questions
--------------------------

Will I need to backup and restore any data?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

The migration will happen automatically, but your administrator should perform a backup prior to the upgrade.

Why are there so many boards on my sidebar?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> We removed workspaces, so if you were a member of many boards prior to migration, they will now all appear under the same sidebar. To help you organize your boards better, you can collapse categories, hide boards, and/or move them to other categories.

Reviewer: ```suggestion
We removed workspaces, so if you were a member of many boards prior to migration, they will now all appear under the same sidebar. To help you organize your boards better, you can collapse categories, hide boards, delete old boards, and/or move them to other categories.
```

How do I find my boards now?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Use the search box (CMD+K/CTRL+K)

Reviewer: ```suggestion
Use the search box (CMD+K/CTRL+K).
```

How do I create a new sidebar category?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Select the **…** menu next to the “Boards” (default) category, and select **Create New Category**.

Reviewer: ```suggestion
Select the **…** menu next to the “Boards” (default) category or any other existing category, and select **Create New Category**.
```

How do I move a board to another category?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Select the **…** menu next to the board’s name in the sidebar, and select **Move To…**.

How do I remove a board from the sidebar?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Select the **…** menu next to the board’s name, and select **Hide board**.

Reviewer: ```suggestion
Select the **…** menu next to the board’s name, and select **Hide board**, or if you're an admin of the board, you can delete the board by selecting **Delete board** from the same menu.

**Please note**: Hiding the board only removes the board from _your_ sidebar. You can add the board back to your sidebar using the search box (CMD+K/CTRL+K). Deleting the board permanently removes the board from the sidebar of all board members.
```

What happens if I delete a category with boards underneath it?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

The boards will be moved to the default “Boards” category. Other users with the same category name will not be impacted.

How do I create a public custom template?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

On the template editor, select **Share** and set the team role to **Editor**.

Help, I’ve got a problem not covered here, what do I do?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Please report any issues to the `public Focalboard channel <https://community.mattermost.com/core/channels/focalboard>`_. We’re here to help!

Reviewer: ```suggestion
Please report any issues to the public `Focalboard channel <https://community.mattermost.com/core/channels/focalboard>`_ or [file a bug](https://github.com/mattermost/focalboard/issues/new?assignees=&labels=Bug%2C+Triage&template=bug_report.md&title=Bug%3A+) on Github. We’re here to help!
```
